src/components/SuggestedValidators/ExpectedReturns.js

```diff
@@ -1,18 +1,23 @@
 import React from "react";
-import { Box, Heading, Text, Tooltip, Icon } from "@chakra-ui/core";
+import { Route } from "react-router-dom";
+import { Box, Button, Link, Heading, Text, Tooltip, Icon } from "@chakra-ui/core";
+import CountUp from "react-countup";
+import CustomButton from "../CustomButton"
 
 type ExpectedReturnsProps = {
 	returns: float,
-	currency: string
+	currency: string,
+	button: bool,
```

Comment:
IIRC, in JS, the data type is "boolean", not "bool"

@AbhinavThukral97 Could you please confirm this? I'm unfamiliar with how the "type" module works

Reply:
It's boolean and is changed.